pkg/entrypoint/entrypointer.go

```diff
@@ -29,6 +29,11 @@ import (
 	"go.uber.org/zap"
 )
 
+//RFC3339 with millisecond
+const (
+	TimeFormat = "2006-01-02T15:04:05.000Z07:00"
```

Comment:
no need to export this const i dont think?  `timeFormat`?

Reply:
Done, thanks